racket/collects/ffi/unsafe.rkt

```diff
@@ -1410,10 +1410,11 @@
 ;; ----------------------------------------------------------------------------
 ;; Struct wrappers
 
-(define (compute-offsets types alignment declared)
+(define* (compute-offsets types [alignment #f] [declared '()])
```

Comment:
A public function should check arguments. I know that many functions here don't, because the original author didn't include them, but we've been trying to fill them in.

Reply:
Fair. I just didn't because I noticed none of the other ones did so I suspected there was a reason for that. I'll fix that. Thanks.